android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/RegisterViewModel.kt

```diff
@@ -166,10 +166,10 @@ constructor(
     when (event) {
       // Search using name or patient logicalId or identifier. Modify to add more search params
       is RegisterEvent.SearchRegister -> {
-        if (event.searchText.isEmpty()) {
+        if (event.searchQuery.isBlank()) {
```

Comment:
We need both checks here. You can create an extension in the `StringExtension.kt` file `isEmptyOrBlank()`